nautobot_circuit_maintenance/forms.py

```diff
@@ -7,7 +7,6 @@
 from django_filters.widgets import BooleanWidget
 from nautobot.circuits.models import Circuit, Provider
 from nautobot.core.forms import (
```

Comment:
While you're touching this, all of the `nautobot.core.forms` imports below can be replaced with `nautobot.apps.forms` as well.